assets/tests/components/__snapshots__/minimalLadderPage.test.tsx.snap

```diff
@@ -1,173 +1,35 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`RouteLadders renders a route ladder 1`] = `
+exports[`MinimalLadderPage renders the preset selection page 1`] = `
 <DocumentFragment>
   <div
-    class="c-ladder-page__tab-bar-and-ladders"
+    class="c-minimal-ladder-page"
   >
-    <div
-      class="c-route-ladders"
-      data-testid="route-ladders-div"
+    <h3>
+      Select a preset to display
+    </h3>
+    <a
+      class=""
+      href="/minimal/abcdef"
     >
-      <div
-        class="c-route-ladder__header"
+      <button
+        class="c-minimal-ladder-page__button btn btn-primary"
+        type="button"
       >
-        <div
-          class="c-route-ladder__close-button-container"
-        >
-          <button
-            aria-label="Close"
-            class="c-close-button c-close-button--large c-close-button--darker"
-          >
-            <span
-              aria-hidden="true"
-              aria-label=""
-              role="img"
-            >
-              <svg />
-            </span>
-          </button>
-        </div>
-        <div
-          class="c-route-ladder__route-name"
-        >
-          1
-        </div>
-      </div>
-      <div
-        class="c-route-ladder__controls"
-      >
-        <button
-          class="c-route-ladder__reverse"
-        >
-          <span
-            class="c-route-ladder__reverse-icon"
-          >
-            <svg />
-          </span>
-          Reverse
-        </button>
-      </div>
-      <div
-        class="c-ladder"
-        style="width: 184px;"
+        Preset 1
+      </button>
+    </a>
```

Comment:
issue: Acording to the anchor element's "Permitted content":

> Permitted content:
>> Transparent, except that no descendant may be interactive content or an <a> element, and no descendant may have a specified tabindex attribute. 
>
> -- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#technical_summary

I believe a `button` is ["interactive content"](https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories#interactive_content) and I think we're really only using this for styling?

What if you put `btn btn-primary` on the `<a>`?